test/specs/collections/Table/TableHeader-test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import * as common from 'test/specs/commonTests'
+
+import TableHeader from 'src/collections/Table/TableHeader'
+
+describe('TableHeader', () => {
+  common.isConformant(TableHeader)
+  common.rendersChildren(TableHeader)
+
+  it('renders as a thead by default', () => {
+    shallow(<TableHeader />)
+      .should.have.tagName('thead')
+  })
+
+  describe('fullWidth', () => {
```

Comment:
Done, including some other refactors that were pending. See https://github.com/TechnologyAdvice/stardust/pull/548.  In short, the final common test argument is now a config object instead of `requiredProps`:

```jsx
common.propKeyOnlyToClassName(FormField, 'required', {
  requiredProps: { label: '' },
})

common.propKeyOnlyToClassName(Table, 'fullWidth', {
  className: 'full-width',
})

common.propKeyOnlyToClassName(Table, 'singleLine', {
  className: 'single line',
})
```